src/bootstrap/test.rs

```diff
@@ -870,6 +870,13 @@ impl Step for RustdocGUI {
             .arg(out_dir)
             .arg("--tests-folder")
             .arg(builder.build.src.join("src/test/rustdoc-gui"));
+        for test_arg in builder.config.cmd.test_args() {
+            if ["--debug", "--no-headless", "--show-text"].iter().any(|a| *a == test_arg) {
+                command.arg(test_arg);
+            } else {
+                panic!("Unknown argument: `{}`", test_arg);
```

Comment:
We generally don't filter arguments in *bootstrap* -- why is this logic not in the script being executed?

Reply:
It is as well, it's true that it's duplicated though... Removing it.